auto_submit/lib/validations/approval.dart

```diff
@@ -117,18 +123,22 @@ class Approver {
       // reviews and will keep them all so the same person can provide two
       // reviews and bypass the two review rule. We make an _approvers
       // contains check to make sure this does not happen.
-      if (state == APPROVED_STATE && !_approvers.contains(authorLogin)) {
+      if (state == APPROVED_STATE && !_reviewAuthors.contains(authorLogin)) {
         _approvers.add(authorLogin);
         if (_remainingReviews > 0) {
           _remainingReviews--;
         }
-        _changeRequestAuthors.remove(authorLogin);
-      } else if (state == CHANGES_REQUESTED_STATE) {
+        //_changeRequestAuthors.remove(authorLogin);
+        //TODO this does not work since we are tracking two different sets.
```

Comment:
Nit: Space following `//`.
Nit: does the comment need a re-phrase?

Reply:
Ah no, my mistake. I forgot to remove this.